### ISSUE TITLE
Format native blocks - Block Category: Text

### ISSUE THREAD

**Issue by SteveRyan-ASU** [asu-ke-web-services/UDS-WordPress-Theme#81]
Stemming from a block planning conversation between the KE team and the Engineering team, we came up with the following punch list of stuff to do regarding the native blocks that ship with WordPress. The majority of this work involves some testing within the default template (sidebar) to make sure that they work as expected out of the box. 

Here's what we determined:

### Block Category: Text
Keep: paragraph, list, code, table, classic block, header
Remove: preformatted, verse, pullquote

#### Pull quote vs blockquote
We only need one of these two blocks for right now. Recommend that we turn off the **pull quote** block for now as the formatting for the pull quote is essentially a `blockquote` element wrapped with a specific wrapper for WordPress.

The blockquote element within UDS has several layers of color combinations possible. The recommended classes for these color combinations within the UDS-Bootstrap project are... not great... so we may be better off reinventing the wheel a little bit within our theme if that's easier. 

#### Header highlight classes
Cross-reference this list with a separate issue opened to add the `<span class="highlight-gold">` classes to the middle of a header. That would be issue #53.

**Comment by SteveRyan-ASU** [comment]
Quoting a discussion from PR #88, we perhaps should consider the following actions when thinking about registering/unregistering blocks. (I had no idea that this was possible via JS initially.)
 
> ### If we think we will get into a situation in which we would want to restrict blocks to a specific page template, then the JS required gets immediately more complicated.
> * See this article for the full description of why: https://webservices.ufhealth.org/2020/07/08/how-to-restrict-blocks-to-specific-page-templates-in-the-wordpress-gutenberg-editor/
> * TL;DR: Since the page template can be switched "on the fly" from within the editor, there are LOTS of things we should potentially do to check the state of the allowed/not-allowed blocks upon the user clicking the dropdown.
> 
> ### The process for unregistering blocks from the editor via PHP appears a lot easier. However, there is a catch.
> * Here's the filter: https://developer.wordpress.org/reference/hooks/allowed_block_types/
> * The second parameter of that filter also allows you to reference the allowed block types list against a specific `wp_post` object.
> * The `wp_post` object contains references to individual pages, page slugs or specific post types. But not to the page template within the same post type.
> 
> ### What I'm wondering is the following:
> * Should we consider moving the `landing.php` template out of the native post type `pages`.
> * ... and instead, create a custom post type called `Landing` with the same intent & markup as the landing.php template.
> * ... so that this whole process of limiting blocks to specific regions of WordPress becomes easier.

**Comment by SteveRyan-ASU** [comment]
Closing this issue. PR submitted which limited the text blocks to the pattern identified above.
Issue #115 addresses the mentioned comments within this issue as well.
